apps/backend/api/src/app/images/images.service.ts

```diff
@@ -1,31 +1,43 @@
 import { Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Image } from '../../entities/image.entity';
-import { Repository } from 'typeorm';
-// import { CreateImageDto } from './dto/create-image.dto';
-// import { UpdateImageDto } from './dto/update-image.dto';
+import { Repository, UpdateResult } from 'typeorm';
+import { CreateImageDto } from './dto/create-image.dto';
+import { UpdateImageDto } from './dto/update-image.dto';
 
 @Injectable()
 export class ImagesService {
   constructor(
     @InjectRepository(Image) private imagesRepository: Repository<Image>,
   ) {}
-  findAllByCategory(category: string) {
+
+  async create(createImageDto: CreateImageDto): Promise<Image> {
+    return await this.imagesRepository.save(createImageDto);
```

Comment:
Quand tu as un `return await` le `await` n'est pas obligatoire
```suggestion
    return this.imagesRepository.save(createImageDto);
```